terraform/deploy_new_portscan_instance.sh

```diff
@@ -53,8 +53,12 @@ function redeploy_instances {
   for index in $(seq "$1" "$2")
   do
     # Check the list of instances and get the ID of the index we are working
-    # on for this iteration
-    nmap_instance_ids+=("$(echo "$portscanner_ids_json" | jq --raw-output ".[] | select(.index == $index) | .id")")
+    # on for this iteration and add it to the array of IDs if found
+    instance_id="$(echo "$portscanner_ids_json" | jq --raw-output ".[] | select(.index == $index) | .id")"
```

Comment:
Is there any output if the instance ID is not found?  It would be nice if there was some sort of user feedback for when this situation arises.

Reply:
There was not, but I have added it in 06d6ac1. Nice catch!

Reply:
Thanks - looks good!